packages/client/src/createMinimalClient.ts

```diff
@@ -52,10 +52,13 @@ export function createMinimalClient(
       throw new Error(`Invalid stack URL: ${baseUrl}${hint}`);
     }
   }
-
+  const processedBaseUrl = new URL(baseUrl);
+  processedBaseUrl.pathname += processedBaseUrl.pathname.endsWith("/")
```

Comment:
If we don't add `/` at the end of the baseUrl, we lose the last parameter in the URL when appending to the URL to hit APIs